attractions/rollup.config.js

```diff
@@ -31,11 +58,17 @@ export default [
     plugins: [
       svelte({
         customElement: true,
-        preprocess: autoPreprocess({
-          scss: { includePaths: ['./'] },
-        }),
+        preprocess: [
+          prependTagOption(icons),
```

Comment:
Shouldn't this be commented out for the time being?

Reply:
Why? Do you not want the custom elements build at all?
This is what fixes the warnings. There should no longer be any warnings, and the custom elements still build as they did

Reply:
I don't get how this fixes the warnings. Just because this code is added artificially and therefore not checked by the checker?

Reply:
Yes. It is added only when we are building, not when the code is imported from somewhere else